nanoFramework.M5Core2.nuspec

```diff
@@ -44,6 +45,7 @@
       <dependency id="nanoFramework.System.IO.Ports" version="1.0.3-preview.6" />
       <dependency id="nanoFramework.System.Numerics" version="1.0.259" />
       <dependency id="nanoFramework.System.Math" version="1.4.3" />
+      <dependency id="nanoFramework.System.Threading" version="1.0.3" />
```

Comment:
It's a dependency of System.Net doesn't need to be here.
(yes, I'm aware it's on most of the others, but it doesn't need either)